src/main/java/com/michelin/ns4kafka/services/AccessControlEntryService.java

```diff
@@ -271,6 +271,14 @@ public List<AccessControlEntry> findAllForCluster(String cluster) {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Find all the ACLs on all clusters
+     * @return A list of ACLs
+     */
+    public List<AccessControlEntry> findAll() {
```

Comment:
Add unit tests for this new method (see: [TopicServiceTest](https://github.com/michelin/ns4kafka/blob/master/src/test/java/com/michelin/ns4kafka/services/TopicServiceTest.java#L916) for example)